### ISSUE TITLE
I don't know why? But I can't reproduce the result by your code from the scratch train.

### ISSUE THREAD

**Issue by GuideWsp** [junpan19/RRN#3]
If there are some tricks that the code not contain?

**Comment by Shuyun-Wang** [comment]
> If there are some tricks that the code not contain?

I have try once,the finial result of average PSNR is 27.509897.It dosn't fit to the result of the paper give.Now i think that the additional trick maybe is the input? Paper show that the input shape is 64 x 64 ,but the train model haven't give any code of this. And there exist a bicubicu.py in the github, i am reading this ,maybe will give you some help.

**Comment by GuideWsp** [comment]
> > If there are some tricks that the code not contain?
> 
> I have try once,the finial result of average PSNR is 27.509897.It dosn't fit to the result of the paper give.Now i think that the additional trick maybe is the input? Paper show that the input shape is 64 x 64 ,but the train model haven't give any code of this. And there exist a bicubicu.py in the github, i am reading this ,maybe will give you some help.

 If possible, we can talk it by wechat. My wechat is “AIWalker-Happy”。

**Comment by hlx-d** [comment]
Have you reproduced the results of the paper?

**Comment by Shuyun-Wang** [comment]
> Have you reproduced the results of the paper?

Nope , it still have some error.

**Comment by zdyshine** [comment]
I trained this network structure, but the loss has been fluctuating and not converging. Can anyone tell me the possible reason？

**Comment by zdyshine** [comment]
> If there are some tricks that the code not contain?



> If there are some tricks that the code not contain?

Is it convenient to add friends to discuss？：Q:690866816

**Comment by GuideWsp** [comment]
> 690866816

you can add my wechat:AIWalker-Happy, I hardly use qq。

**Comment by uriyal-rnd** [comment]
> I trained this network structure, but the loss has been fluctuating and not converging. Can anyone tell me the possible reason？

Got the same issue, the loss doesn't converge and keep fluctuating.
I've tried to fine-tune the provided pre-trained model (model/RRN-5L.pth), lowering the LR to 1e-5 and training the model for a single epoch on the Vimeo-90 dataset with online cropping from the center 256x256 patches.
It looks like the loss is keep on fluctuating around 1500,
after finetuning, I tried to re-run the test script on the test data (vid4) with the new weights and the results are:
==> Average PSNR = 27.304463
==> Average SSIM = 0.834354

FYI, I've verified that the provided pre-trained model (model/RRN-5L.pth) on the test data, use it without any modification (without the fine-tuning) and it approx. matched the published results (27.69/0.8488):
==> Average PSNR = 27.391460
==> Average SSIM = 0.839081

**Comment by lunaSXF** [comment]
> I trained this network structure, but the loss has been fluctuating and not converging. Can anyone tell me the possible reason？

MEE TOO,have you solved this problem?

**Comment by Afeiaa** [comment]
Has anyone solved this problem?
